extractor/image/image.go

```diff
@@ -65,7 +66,22 @@ func NewDockerImage(ctx context.Context, imageName string, option types.DockerOp
 	return img, func() {}, nil
 }
 
+// TODO: rename NewDockerArchiveImage to NewArchiveImage
 func NewDockerArchiveImage(fileName string) (v1.Image, error) {
```

Comment:
`NewArchiveImage` sounds good to me.

```suggestion
func NewArchiveImage(fileName string) (v1.Image, error) {
```

Reply:
Done https://github.com/aquasecurity/fanal/pull/101/commits/d9baf353482f82019971d255076eeeb63af2d5ec

Reply:
Can you also rename the test accordingly? https://github.com/aquasecurity/fanal/pull/101/files#diff-525dd35201b72b43ad63b85dd2720ba6L105

`func TestNewDockerArchiveImage(t *testing.T) {`
to 
`func TestNewArchiveImage(t *testing.T) {`

Reply:
Thanks. I've fixed it.
https://github.com/aquasecurity/fanal/pull/101/commits/9163201742aada50207d588bc5d2d5366270140a